src/Products/GenericSetup/ZCTextIndex/exportimport.py

```diff
@@ -16,11 +16,10 @@
 from BTrees.IOBTree import IOBTree
 from BTrees.Length import Length
 from BTrees.OIBTree import OIBTree
-from zope.component import adapts
-
 from Products.ZCTextIndex.interfaces import IZCLexicon
 from Products.ZCTextIndex.interfaces import IZCTextIndex
 from Products.ZCTextIndex.PipelineFactory import element_factory
+from zope.component import adapts
```

Comment:
This is broken. That change should not have been made. Looks like the isort policy doesn't work correctly.